Jump_Search/Jump_Search.cpp

```diff
@@ -0,0 +1,58 @@
+// Cpp program to implement Jump Search 
+#include <bits/stdc++.h> 
+using namespace std; 
+
+int jumpSearch(int arr[], int element, int number_of_element) 
+{ 
+	// Finding block size to be jumped 
```

Comment:
Fix indentation.

Reply:
What will i do here ?? Its indented i guess

Reply:
It is indented with 8 spaces. Change it to 4 spaces.

Reply:
done